InstallerLib/Installer.cs

```diff
@@ -124,15 +126,33 @@ public async Task<bool> Install()
                     var startMenu = Path.Join(startMenuFolder, shortcut);
                     var startup = Path.Join(Environment.GetFolderPath(Environment.SpecialFolder.Startup), shortcut);
                     var uninstaller = Path.Join(startMenuFolder, "Uninstall.lnk");
-                    var otd = Path.Join(InstallationInfo.Current.InstallationDirectory.FullName, Launcher.AppName + ".exe");
+                    var otd = Path.Join(InstallationDirectory.FullName, Launcher.AppName + ".exe");
 
                     CleanShortcuts();
                     Directory.CreateDirectory(startMenuFolder);
-                    CreateShortcut(desktop, updater);
-                    CreateShortcut(startMenu, updater);
-                    CreateShortcut(startup, updater, "--minimized", startMinimized: true);
-                    CreateShortcut(uninstaller, updater, "--uninstall");
-                    CreateShortcut(InstallationInfo.Current.OTDProxy.FullName, otd, startMinimized: true);
+                    Shortcut.Save(desktop, updater);
+                    Shortcut.Save(startMenu, updater);
+                    Shortcut.Save(startup, updater,
+                        Minimized: true);
+                    Shortcut.Save(uninstaller, updater,
+                        Arguments: "--uninstall");
+                    Shortcut.Save(InstallationInfo.Current.OTDProxy.FullName, otd,
+                        WorkingDirectory: InstallationDirectory.FullName,
+                        Minimized: true);
+
+                    var otdRegistry = new Registry(@"HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\OpenTabletDriver")
```

Comment:
I'd define the key as a constant to avoid accidental errors in the future
```cs
private const string UNINSTALL_REG_KEY = @"HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\OpenTabletDriver";
```

Reply:
Requested change pushed